client/css/style.css

```diff
@@ -374,8 +370,7 @@ kbd {
 }
 
 #viewport.rt #chat .sidebar {
-	-webkit-transform: translate3d(180px, 0, 0);
-	transform: translate3d(180px, 0, 0);
+	right: -180px;
```

Comment:
From the links you posted in PR description, I understand that `right` would need a repaint as opposed to `transform`. Is that what `will-change: transform;` below is trying to prevent?
Also, how does `will-change: transform;` work if you removed the `transform` property above?

Sorry, still trying to understand how all of this works...